test/suites/zombie_tanssi_relay_eth_bridge/test_zombie_tanssi_relay_eth_bridge.ts

```diff
@@ -423,6 +453,85 @@ describeSuite({
 
         it({
             id: "T06",
+            title: "Rewards are claimable",
+            test: async () => {
+                // Find the first era index claimable
+                const currentEra = (await relayApi.query.externalValidators.activeEra()).unwrap().index;
+
+                let eraToAnalyze = currentEra.toNumber();
+                console.log(await operatorRewardContract.eraRoot(eraToAnalyze));
+
+                // Try to find latest reported era
+                // eraRoot returns a struct of 4 items, the 3rd of which is the era root
+                // we try to retrieve the first non-default one
+                while (
+                    (await operatorRewardContract.eraRoot(eraToAnalyze))[3] ===
```

Comment:
Does indexing start from 1?
You mentioned that you are looking for 3rd argument, so shouldn't it be with index 2?

Reply:
yes sorry, 5 items and I am looking for 4th, will updaget